deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Configured as part of the DigitalOcean 1-Click Image build process
+
+myip=$(hostname -I | awk '{print$1}')
+cat <<EOF
+********************************************************************************
+
+Welcome to VictoriaMetrics server!
+To keep this server secure, the UFW firewall is enabled.
+All ports are BLOCKED except 22 (SSH), 80 (HTTP), and 443 (HTTPS), 8428 (VictoriaMetrics HTTP), 8089 (VictoriaMetrics Influx),
+4242 (VictoriaMetrics OpenTSDB), 2003 (VictoriaMetrics Graphite)
+
+In a web browser, you can view:
+ * The VictoriaMetrics Quickstart guide: https://kutt.it/1click-quickstart
+
+On the server:
+  * The default VictoriaMetrics root is located at /var/lib/victoria-metrics-data
+  * VictoriaMetrics is running on ports: 8428, 8089, 4242, 2003 and they are bound to the local interface.
+
+********************************************************************************
+  # This image includes version VM_VERSION of VictoriaMetrics.
```

Comment:
```suggestion
  # This image includes version $VM_VERSION of VictoriaMetrics.
```

Reply:
There is a default template file `99-one-click.tpl`, which is copied to 99-one-click where the `VM_VERSION` is replaced with the version passed to make - see https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3142/files/36a6bdbc5f8e6590e4f3002b5198535ae5f45a71#diff-454031dadedbca78b6d08f2e42cba83bbcc8999ac99778e38dca05e6b19cd5a3

Reply:
Ah, ok, got it.